Kitodo/diagrams/Example_Workflow.bpmn20.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you add a licence header here?

Reply:
It is generate file - do they also need license header?